src/pages/Onboarding/IntroLandingPage.tsx

```
@@ -0,0 +1,237 @@
+      window.removeEventListener("resize", onResize);
+    };
+  }, [idx, centerActive]);
+
+  const SWIPE_THRESHOLD = 40;
+  const onTouchStart = (e: React.TouchEvent) => {
+    touchStartX.current = e.touches[0].clientX;
+  };
+  const onTouchEnd = (e: React.TouchEvent) => {
+    if (touchStartX.current == null) return;
+    const dx = e.changedTouches[0].clientX - touchStartX.current;
+    if (Math.abs(dx) > SWIPE_THRESHOLD) dx > 0 ? prev() : next();
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)